core/src/test/resources/sql/functions.iq

```diff
@@ -184,6 +185,46 @@ SELECT CONCAT('a');
 
 !ok
 
+SELECT CONCAT('a', 'b', 'c');
++--------+
+| EXPR$0 |
++--------+
+| abc    |
++--------+
+(1 row)
+
+!ok
+
+SELECT CONCAT('a', x'61');
++--------+
+| EXPR$0 |
++--------+
+| 6161   |
++--------+
+(1 row)
+
+!ok
+
+SELECT CONCAT(x'61', x'62');
++--------+
+| EXPR$0 |
++--------+
+| 6162   |
++--------+
+(1 row)
+
+!ok
+
+SELECT CONCAT('a', x'61', null);
```

Comment:
can you please add some tests that
- use implicit casts (e.g., integer arguments); if the validator accepts these programs, they should run 
- use invalid arguments, if there are any, checking for an error

Reply:
OK，had added implicit casts quidem test and invalid aruguments which would error checking.